Cargo.toml

```diff
@@ -11,8 +11,10 @@ description = "A more memory efficient replacement for serde_json::Value"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
 tracing = ["mockalloc/tracing"]
+thread_unsafe = ["ctor"]
```

Comment:
Features should be additive, so the way this should be done is to have a `thread_safe` feature which is in `default_features` for backwards compatibility.

Reply:
Also this feature doesn't need to have any relation to the `ctor` crate.

Reply:
This is why I added `thread_unsafe` which is optional the default is still not `thread_unsafe` to make sure no backward.

Reply:
I'm using the `ctor` as a way to assure a `lazy` init on the hashmap

Reply:
> This is why I added thread_unsafe which is optional the default is still not thread_unsafe to make sure no backward.

Right, but that's not an additive feature, and features must be additive.

If we have two crates A & B, where A enables the `thread_unsafe` feature, but B doesn't because it uses the crate across multiple threads, then a third crate C may depend on both A & B, and fail to compile because B won't compile when the `thread_unsafe` feature is enabled.

Reply:
> I'm using the ctor as a way to assure a lazy init on the hashmap

It's still not necessary to make this Cargo.toml change - if the `ctor` feature is enabled then it will be used for lazy init, if it's not then it won't.